app/src/main/java/com/example/findu/presentation/ui/report/WitnessReportFragment.kt

```
@@ -1,506 +1,251 @@
-            submitList(reportViewModel.imageUriList.value)
-        }
-
-        with(binding.rvWitnessReportImages) {
-            adapter = reportImageAdapter
-            layoutManager = LinearLayoutManager(
-                context,
-                LinearLayoutManager.HORIZONTAL,
-                false
-            )
+            startActivity(intent)
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)